sample-compose/app/src/main/java/co/nimblehq/sample/compose/ui/AppNavigation.kt

```diff
@@ -0,0 +1,51 @@
+package co.nimblehq.sample.compose.ui
+
+import androidx.compose.runtime.Composable
+import androidx.navigation.*
+import androidx.navigation.compose.*
+import co.nimblehq.sample.compose.ui.screens.home.HomeComposeScreen
+import co.nimblehq.sample.compose.ui.screens.second.SecondScreen
+
+@Composable
+fun AppNavigation(
+    navController: NavHostController = rememberNavController(),
+    startDestination: String = AppDestination.Home.destination
+) {
+    NavHost(
+        navController = navController,
+        startDestination = startDestination
+    ) {
+        composable(AppDestination.Home) {
+            HomeComposeScreen(
+                navigator = { destination -> navController.navigate(destination) }
+            )
+        }
+
+        composable(AppDestination.Second) {
+            SecondScreen(
+                navigator = { destination -> navController.navigate(destination) },
+                id = it.arguments?.getString(KEY_ID).orEmpty()
+            )
+        }
+    }
+}
+
+private fun NavGraphBuilder.composable(
+    destination: AppDestination,
+    deepLinks: List<NavDeepLink> = emptyList(),
+    content: @Composable (NavBackStackEntry) -> Unit
+) {
+    composable(
+        route = destination.route,
+        arguments = destination.arguments,
+        deepLinks = deepLinks,
+        content = content
+    )
+}
+
+private fun NavHostController.navigate(destination: AppDestination) {
+    when (destination) {
+        is AppDestination.Up -> popBackStack()
```

Comment:
@luongvo Should it be `navigateUp()` instead of `popBackStack()` 🤔 
```suggestion
        is AppDestination.Up -> navigateUp()
```

Reply:
Fixed in https://github.com/nimblehq/android-templates/pull/367/commits/292e3157b647217bffd88bcba8b6874063dc981e